torch/_inductor/kernel/mm_common.py

```diff
@@ -19,18 +19,27 @@ def triton_config(num_stages, num_warps, **kwargs):
 
 
 def filtered_configs(
-    m: int, n: int, k: int, configs: List[Tuple[int, int, int, int, int]]
+    m: int,
+    n: int,
+    k: int,
+    configs: List[Tuple[int, int, int, int, int]],
+    has_int8_tensor=False,
 ):
     """Heuristic to shrink configs when they are bigger than the input size"""
-    m = max(next_power_of_2(V.graph.sizevars.size_hint(m)), 16)
-    n = max(next_power_of_2(V.graph.sizevars.size_hint(n)), 16)
-    k = max(next_power_of_2(V.graph.sizevars.size_hint(k)), 16)
+
+    # According to https://github.com/openai/triton/issues/2156#issuecomment-1695897424
```

Comment:
BTW, it could be 16x32 if you want to try to improve perf a bit :).

Reply:
So that means m can be 16 but n and k have to be at lease 32 for int8?
Since we have tl.tensor with shape [m, k], [k, n], [m, n] in the triton kernel.

Reply:
> So that means m can be 16 but n and k have to be at lease 32 for int8?

k must be >= 32 but n and m can be >= 16 if both `a` and `b` in `axb` are not transposed.